tracks/rust/exercises/bracket-push/mentoring.md

```diff
@@ -1,20 +1,58 @@
 ### Concepts
 
 - stack or recursion
+- `HashMap`s and `HashSet`s
 
 ### Reasonable solutions
 
 A reasonable solution should do the following:
 
+- Keep track of the associations between specific brackets using a `HashMap`, a closure, or something of the like. 
```

Comment:
This is bad advice, leading to way too heavy an implementation. All that is necessary is a stack.